config.toml

```diff
@@ -0,0 +1,24 @@
+baseURL = ""  # This is populated by the github deploy workflow and is equal to "<domainName>/<version|latest>"
+languageCode = "en-us"
+title = "Apache Iceberg"
+theme= "hugo-book"
+
+[params]
+  BookLogo = "img/iceberg-logo-icon.png"
+  description = "The open table format for analytic datasets."
+  versions.iceberg = "" # This is populated by the github deploy workflow and is equal to the branch name
+  versions.nessie = "0.15.1"
+  latestVersions.iceberg = "0.12.1"  # This is used for the version badge on the "latest" site version
+
+[[params.social]]
+  title = "Mailing Lists"
+  icon = "envelope-o"
+  url = "https://iceberg.apache.org/#community/#mailing-lists"
```

Comment:
This link isn't correct. The hugo site won't use `#community`, it will use the old paths with no `#`. That's a good thing, since that matches the URLs that we've supported for a few years.